src/codegen.cpp

```diff
@@ -5964,7 +5961,7 @@ static void init_julia_llvm_env(Module *m)
         &pow,
```

Comment:
This breaks compilation here.

What's the reason again that we can't do the `static_cast` unconditionally? Including `<math.h>` in C++ code causes `using std::pow;` which provides additional overloads for `float` and `long double`.